src/test/java/com/hotels/bdp/circustrain/bigquery/extraction/BigQueryDataExtractionServiceTest.java

```diff
@@ -98,6 +109,12 @@ public void cleanupWhenDeletionFailsDoesntThrowException() {
     Bucket bucket = mock(Bucket.class);
     when(bucket.delete()).thenReturn(true);
     when(storage.get(anyString())).thenReturn(bucket);
+    Blob blob = mock(Blob.class);
+    List<Blob> blobs = ImmutableList.of(blob);
```

Comment:
If want to the test to be a bit more realistic we should have more than one blob in the list and then verify that delete was called for all of them.